zulipterminal/ui_tools/views.py

```diff
@@ -879,10 +879,14 @@ def __init__(self, controller: Any,
 
 
 class AboutView(PopUpView):
-    def __init__(self, controller: Any, title: str, *,
-                 zt_version: str) -> None:
-        contents = [
-            ('Version', [('Zulip Terminal', zt_version)]),
+    def __init__(self, controller: Any, title: str, *, zt_version: str,
+                 server_version: str, server_feature_level: str) -> None:
+        contents = ([('', [('Server feature level', server_feature_level)])]
+                    if server_feature_level else [])
+
+        contents += [
+            ('Version', [('Zulip Terminal', zt_version),
+                         ('Server', server_version)]),
```

Comment:
I think it'd be better to group the server properties together; when I merged the first two commits I set the `Zulip Terminal` category to be in `Application`, and so I'd suggest we have a `Server` category with the extra details.